.tflint.hcl

```diff
@@ -1,5 +0,0 @@
-plugin "aws" {
-    enabled = true
-    version = "0.17.1"
```

Comment:
Did you try updating the version? it also works
I'm leaning toward working with fixed versions rather than always using the latest (I think that happens when removing this file)
version = "0.28.0"

Reply:
We don't need tflint unless explicitly required, but this is not the case, so it's safe to delete it and let workflow handle it.

Reply:
Gotcha